docs/backplane/requirements/acs/10-acs-managed-service.md

```diff
@@ -32,5 +43,38 @@ Permissions at cluster scope.
 * get pods and pod logs
 * view deployments
 * get deployments
+* view routes
+* view egress firewalls
+* view roles and rolebindings
+* patch persistentvolumeclaims
+
+ACS team needs to inspect the `cloud-service-sensible-declarative-configs` secret
+
+* view secret `cloud-service-sensible-declarative-configs`
+
+ACS team needs to be able to delete a namespace
```

Comment:
Namespace is a cluster scope resource, so delete is pretty broad and in the wrong place for requirements. What is the reason delete is needed?  ACS permissions for creating namespace is not granted so any namespace created is done through some controller or config management.

Reply:
I see. This was the SOP for https://gitlab.cee.redhat.com/stackrox/acs-cloud-service/runbooks/-/blob/master/sops/dp-024-lost-or-invalid-tenant-secrets.md?ref_type=heads . We don't have any controller that allows deleting namespaces.

Reply:
In the SOP a force-reconcile step is an option.  If that is done without deleting the namespace will the state be restored?  Deletion of the namespace looks like a stop-gap to cover a gap in reconciliation, which should just restore a deleted secret if it is managed.

How often is this SOP executed?  I.e. is it a "I need to do this every day" kind of permission.

Reply:
Ah, no, and thank god this is not everyday. Would this be a candidate for a managed script?

Reply:
I will remove the ask for now, until we find a compliant way

Reply:
I think this should be managed in the force reconcile, no need for any additional in-cluster permissions. And actually, why wouldn't reconcile just recreate the secret if it's missing or reconcile state of resources under management if they are manually changed? If removed this is fine, discussion on how can happen elsewhere.

Reply:
There's 2 reconciliations: 
1. the fleet-manager reconciliation, that creates Centrals for each tenant. There's a cache in there that will skip reconciliation. But it can be "forced"
2. The operator reconciliation, which creates resources for each Central (e.g. deployments, etc.). This one cannot be forced-reconciled. 

It's in the case that there's a problem with 2. that we would like to delete stuff that the operator created, but that are in a bad state. There's no mechanism for us to do that right now apart from click-ops.

Reply:
But I removed the permission and will keep track in jira